sql/core/src/test/resources/sql-tests/results/ansi/datetime.sql.out

```diff
@@ -359,31 +359,31 @@ cannot resolve '1 + (- INTERVAL '2 seconds')' due to data type mismatch: argumen
 -- !query
 select date'2020-01-01' - timestamp'2019-10-06 10:11:12.345678'
 -- !query schema
-struct<subtracttimestamps(DATE '2020-01-01', TIMESTAMP '2019-10-06 10:11:12.345678'):interval>
+struct<subtracttimestamps(DATE '2020-01-01', TIMESTAMP '2019-10-06 10:11:12.345678'):day-time interval>
```

Comment:
Have we finalized the SQL name for the new interval types? How about other databases?

Reply:
Not yet. So far, I took the names for the sub-types from the SQL standard, see https://github.com/apache/spark/pull/31810

Probably, we will need to re-define them when we will implement parsing of interval types from SQL.